lib/kubernetes-deploy/remote_logs.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/container_logs'
+
+module KubernetesDeploy
+  class RemoteLogs
+    attr_reader :container_logs
+
+    def initialize(logger:, parent_id:, container_names:)
+      @logger = logger
+      @parent_id = parent_id
+      @container_logs = container_names.map do |n|
+        ContainerLogs.new(logger: logger, container_name: n, parent_id: parent_id)
+      end
+    end
+
+    def empty?
+      @container_logs.all?(&:empty?)
+    end
+
+    def sync(kubectl)
+      @container_logs.each { |cl| cl.sync(kubectl) }
+    end
+
+    def print_latest
+      @container_logs.each { |cl| cl.print_latest(prefix: @container_logs.length > 1) }
+    end
+
+    def print_all(prevent_duplicate: true)
```

Comment:
What's a use case where someone would want to set `prevent_duplicate` to `false`? (doesn't seem like that's actually being used anywhere (yet?)).

Reply:
There isn't one right now. I added this because of one of your comments on Danny's original PR, something to the effect of preventing duplicate being unexpected default behaviour. I obviously misunderstood your ask though. 😆